store/storetest/preference_store.go

```diff
@@ -434,7 +455,9 @@ func testPreferenceCleanupFlagsBatch(t *testing.T, ss store.Store) {
 		Value:    "true",
 	}
 
-	store.Must(ss.Preference().Save(&model.Preferences{preference1, preference2}))
+	count, err := ss.Preference().Save(&model.Preferences{preference1, preference2})
+	require.Nil(t, err)
+	require.Equal(t, 2, count)
 
 	_, err := ss.Preference().CleanupFlagsBatch(10000)
```

Comment:
This will fix the build error.

```suggestion
	_, err = ss.Preference().CleanupFlagsBatch(10000)
```

Reply:
Updated

Reply:
I'm not able to debug which tests are breaking from logs. Can you tell me how can I check that?

Reply:
To know exactly which error is really causing the test to fail, I normally search for "FAIL" with case sensitivity enabled.

But In this case, I think is a problem with a flaky test. I'm running it again.